pyproject.toml

```diff
@@ -102,8 +102,8 @@ dbt-tags-test-unapply-mps = [
 dbt-tags-test-drop-tags = [
   {cmd = "poe dbt-tags-tagging"},
   {cmd = "dbt run-operation drop_tags --project-dir integration_tests"},
-  {cmd = "dbt run -s verify_if_tags_created_correctly --project-dir integration_tests"},
-  {cmd = "dbt test -s verify_if_drop_tags_correctly --project-dir integration_tests"},
+  {cmd = "dbt run -s verify_if_tags_created_correctly --project-dir integration_tests --vars {'tag_source_columns':false}"},
```

Comment:
I got an error running this command `poe dbt-tags-test-drop-tags`
Error message is as following:

> on-run-end failed, error:
 002003 (02000): SQL compilation error:
Tag 'DBT_TAGS.DBT_THU_COMMON.PII_NULL' does not exist or not authorized.

Reply:
Yes, this is happening because, as part of the test:
1. We delete the tags.
2. Then, we build the model `verify_if_tags_created_correctly`. The issue arises because we are using the `apply_source_column_tags` macro in the on-run-end hook, which tries to apply the tags after the run. However, by that point, the tags have already been deleted by the `dbt run-operation drop_tags`.

To prevent this, I have introduced a variable, `tag_source_columns`, that ensures we don't apply `apply_source_column_tags` in these specific scenarios.

Reply:
Hi @thu-IL , I've made a couple of updates. Could you please test them and let me know if everything works, so I can close the conversation?

Reply:
Thanks @bsuryachandra. I'm ok with the updates on `dbt-tags-test-drop-tags`.

What I was also trying to say is that we have a model named `verify_if_tags_applied_correctly` which is to test if we successfully applied tags as we wanted to. Could you please help revise the model to see if there is anything you need to update there? I guess it would be schema name logic because source tables must be in a different schema than target schema

Reply:
Thanks for pointing that out! I’ve updated the `verify_if_tags_applied_correctly` model to handle this scenario. Could you give it a test and let me know if it looks good?

Reply:
Looks good to me. Thank you very much for your contribution @bsuryachandra